tests/pages/base.py

```diff
@@ -41,3 +46,18 @@ def languages(self):
         def select_language(self, value):
             el = self.root.find_element(*self._language_locator)
             Select(el).select_by_value(value)
+
+    class FamilyNavigation(PageRegion):
+
+        _tertiary_nav_button = (By.ID, 'fxfamilynav-tertiarynav-trigger')
+        _tertiary_nav_menu = (By.ID, 'fxfamilynav-tertiarynav')
+
+        def click_tertiary_menu(self):
+            self.selenium.find_element(*self._tertiary_nav_button).click()
+            Wait(self.selenium, self.timeout).until(
+                expected.visibility_of_element_located(self._tertiary_nav_menu))
+
+        @property
+        def tertiary_menu_visible(self):
+            el = self.selenium.find_element(*self._tertiary_nav_menu)
+            return el.is_displayed()
```

Comment:
Also, where possible I would recommend providing a root element for a `PageRegion`. If you avoid having the button that opens the menu within this region as suggested above, you could use `_root_locator = (By.ID, 'fxfamilynav-tertiarynav')`. This means that any locators within this region are looked up from the scope of this element rather than the entire document. While the performance advantage is negligible, it can help when using locators that are not unique.